compiler/damlc/tests/daml-test-files/InterfaceChoiceCollision.EXPECTED.desugared-daml

```diff
@@ -66,35 +66,35 @@ _choice_TArchive
   = (\ this _ -> DA.Internal.Desugar.signatory this, 
      \ _ _ _ -> pure (), DA.Internal.Desugar.Consuming, 
      DA.Internal.Desugar.None)
-_implements_T_InterfaceChoiceCollision1_Interface :
+_implements_T_InterfaceChoiceCollision1:Interface :
```

Comment:
I changed the separator for qualified names to `:` instead of `_` so it's easier to see what's what. 

(thanks @remyhaemmerle-da for asking these questions)

I think this should be safe since these markers don't make it to LF (see [DA.Daml.LFConversion#1048-1055](https://github.com/digital-asset/daml/blob/qualified-requires-marker/compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs#L1048-L1055)), and there's no risk of collisions because none of {module, template, interface} names can contain colons.